ProjectSettings/DynamicsManager.asset

```diff
@@ -12,4 +12,4 @@ PhysicsManager:
   m_SolverIterationCount: 6
   m_QueriesHitTriggers: 1
   m_EnableAdaptiveForce: 0
-  m_LayerCollisionMatrix: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
```

Comment:
We could always have no physics hands in our demos, and use the documentation to suggest the layer strategy if the user wants to add physics hands.